style.css

```diff
@@ -0,0 +1,108 @@
+/* General CSS Rules */
+:root {
+  --blue: #6070ff;
+}
+
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+a {
+  text-decoration: none;
+  cursor: pointer;
+}
+
+button {
+  cursor: pointer;
+  border: none;
+  background: none;
+}
+
+ul {
+  list-style: none;
+}
+
+/* Navbar Design */
+nav {
+  display: flex;
+  position: fixed;
+  width: 100%;
+  background-color: white;
+  height: 8vh;
+  top: 0;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.links {
+  display: none;
+}
+
+.logo {
+  margin-left: 24px;
+}
+
+.logo a {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: var(--blue);
+}
+
+.hamburger {
+  margin-right: 2vw;
+}
+
+/* Headline Design */
+header {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 680px;
+  background-color: white;
+  margin-top: 8vh;
+  border-bottom-left-radius: 8em;
+  padding: 0 24px 10px 24px;
+  background-image: url(img/header-background.png);
+  background-position: 50% 50%;
```

Comment:
- [ ] I kindly 🙏🙏request that you ensure your project is responsive according to the project [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md), 
- [ ] specifically targeting mobile versions ranging from `375px to 768px.` By optimizing the design for different screen sizes, you will create a more user-friendly and visually appealing experience for your audience. 🥱🥱
- [ ] Your attention to this detail will greatly enhance the overall quality of your project.
![res](https://github.com/MasterLeiva/DavidLeiva-Portfolio/assets/118761303/35405c2a-a681-43e3-bb83-a31439f33517)